docs/romanisim/running.rst

```diff
@@ -15,43 +15,49 @@ The ``romanisim-make-image`` command line interface (CLI) has a number of argume
+    usage: romanisim-make-image [-h] [--bandpass BANDPASS] [--boresight] [--catalog CATALOG] [--config CONFIG] [--date DATE] [--level LEVEL] [--ma_table_number MA_TABLE_NUMBER]
+    [--nobj NOBJ] [--previous PREVIOUS] [--radec RADEC RADEC] [--rng_seed RNG_SEED] [--roll ROLL] [--sca SCA] [--usecrds] [--psftype {None,epsf,galsim,stpsf}]
+    [--webbpsf] [--stpsf] [--truncate TRUNCATE] [--pretend-spectral PRETEND_SPECTRAL] [--drop-extra-dq] [--scale-factor SCALE_FACTOR]
+    [--extra-counts EXTRA_COUNTS [EXTRA_COUNTS ...]]
+    filename
 
     Make a demo image.
 
     positional arguments:
-      filename              output image (asdf)
+      filename              output image (asdf). {} and {bandpass} strings will be automatically populated with detector and bandpass information.
 
     options:
       -h, --help            show this help message and exit
       --bandpass BANDPASS   bandpass to simulate (default: F087)
       --boresight           radec specifies location of boresight, not center of WFI. (default: False)
       --catalog CATALOG     input catalog (ecsv) (default: None)
       --config CONFIG       input parameter override file (yaml) (default: None)
-      --date DATE           UTC Date and Time of observation to simulate in ISOT format. (default: None)
+      --date DATE           UTC Date and Time of observation to simulate in ISOT format. (default: 2026-01-01T00:00:00.000)
       --level LEVEL         1 or 2, for L1 or L2 output (default: 2)
       --ma_table_number MA_TABLE_NUMBER
       --nobj NOBJ
       --previous PREVIOUS   previous simulated file in chronological order used for persistence modeling. (default: None)
       --radec RADEC RADEC   ra and dec (deg) (default: None)
       --rng_seed RNG_SEED
       --roll ROLL           Position angle (North towards YIdl) measured at the V2Ref/V3Ref of the aperture used. (default: 0)
-      --sca SCA             SCA to simulate. Use -1 to generate images for all SCAs; include {} in filename for this mode to indicate where the SCA number should be filled, e.g.
-                            l1_wfi{}.asdf (default: 7)
-      --usecrds             Use CRDS for distortion map (default: False)
-      --stpsf               Use stpsf for PSF (default: False)
+      --sca SCA             SCA to simulate. Use -1 to generate images for all SCAs; include {} in filename for this mode to indicate where the detector number should be filled, e.g.
+                            l1_{}.asdf (default: 7)
+      --usecrds             Use CRDS references. (default: False)
+      --psftype {None,epsf,galsim,stpsf}
+                            Type of PSF generator to use. If None and --usecrds, then "epsf" will be used otherwise "galsim" is used. (default: None)
```

Comment:
```suggestion
                            Type of PSF to use. If None and --usecrds, then "epsf" will be used otherwise "galsim" is used. (default: None)
```